### PR TITLE
Added support for bonus points and account/documents page

### PR DESCRIPTION
**Pull request by Carsten89**
# Pull Request Template

## Description

Added support for bonus points and account/documents page

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php [hunk 1]
```diff
@@ -500,6 +500,56 @@ public function checkoutAction()
...
+            session_start(array(
+                'sessionId' => $sessionId
+            ));
+        }
+
+        $token = $this->Request()->getParam('token');
+        $this->session->offsetSet('sgWebView', true);
+
+        if ($this->webCheckoutHelper->loginAppUser($token, $this->Request())) {
+            $this->redirect('BonusSystem/points');
```
Comment (Stereck) [inline src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php:522]: Only this line differs in every method. Can you please create a general one an parametrize this string for every action in this class? I know this should have been done earlier, but now things adding up =)

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Fixed
 - export of customer group specific prices
 
+## [2.9.91] - 2020-06-24
```
Comment (Stereck) [inline CHANGELOG.md:15]: Please move up, change date so everything under "unreleased" will be part of the release

File: src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php (hunk 1 above)
Comment (Carsten89) [reply src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php:526]: i moved some parts out to a new function

File: CHANGELOG.md (hunk 2 above)
Comment (Carsten89) [reply CHANGELOG.md:15]: done

**Comment by Stereck** [comment]
There are CS issues, please check and fix
https://travis-ci.org/github/shopgate/cart-integration-shopware/jobs/703837763

File: src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php [hunk 3]
```diff
@@ -301,6 +301,24 @@ public function getSettingsAction()
...
 
+    private function loginHelper()
+    {
+        $sessionId = $this->Request()->getParam('sessionId');
+
+        if (isset($sessionId)) {
+            session_write_close();
+            session_id($sessionId);
+            session_start(array(
+              'sessionId' => $sessionId
```
[the remark this answers is not in the document]
Comment (Carsten89) [reply src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php:312]: the code was always like this. so i would keep it as it is

**Pull request merged**